_posts/2022-05-16-zarr-2.12.0a1-release.markdown

```diff
@@ -6,4 +6,76 @@ categories: blog
 permalink: /release-2-12-0-a1/
 ---
 
-Updating soon...
\ No newline at end of file
+Pre-release version `2.12.0a1` of the [Zarr Python package](https://github.com/zarr-developers/zarr-python) has been released! 🎉
```

Comment:
Blogging about a pre-release is primarily focused on getting testing. I could see putting some of the extra points behind, "please see the release announcement" (though the release announcement URL will likely disappear)

Reply:
The link for the release notes is added [here](https://github.com/MSanKeys963/blog-1/blob/main/_posts/2022-05-16-zarr-2.12.0a1-release.markdown#more-information).